### ISSUE TITLE
Missing begin end block for converted code cells

### ISSUE THREAD

**Issue by contra-bit** [adrhill/JupyterPlutoConverter.jl#6]
When a Jupyter notebook cell is converted into a Pluto cell, multiple expressions are added into one cell. 

Given one example, Jupyter cell
```julia
x = 1
y = 2
```

It should not create
```julia
x = 1
y = 2
```

But rather
```julia
begin
    x = 1
    y = 2 
end
```

As one block should remain, one block.

**Comment by fonsp** [comment]
Hi @contra-bit ! This can be achieved with the `wrap_block ` keyword argument. Maybe it was added after you posted the issue... Take a look at the new README for its documentation :)

**Comment by adrhill** [comment]
Maybe `wrap_block` should default to `true`?

**Comment by fonsp** [comment]
Yeah I think so! But I would like to make it a bit smarter, to not just check for multiple lines of code but for multiple expressions... Let me add that check as API to pluto

**Comment by fonsp** [comment]
https://github.com/fonsp/Pluto.jl/pull/3134
